neural_lam/models/ar_model.py

```diff
@@ -473,7 +526,7 @@ def create_metric_log_dict(self, metric_tensor, prefix, metric_name):
         # Check if metrics are watched, log exact values for specific vars
         if full_log_name in self.args.metrics_watch:
             for var_i, timesteps in self.args.var_leads_metrics_watch.items():
-                var = self.config_loader.dataset.var_names[var_i]
+                var = self.data_config.vars_names("state")[var_i]
```

Comment:
There's a few places that still refer to `self.data_config` that I believe should be changed to point to the datastore. Do a search and replace each of these with corresponding usage of datastore.

Reply:
All references to `data_config` are now gone (and replaced with `datastore` as appropriate) in https://github.com/leifdenby/neural-lam/commit/7e46194b91bd2b7c00850bbb925b59e1fa98d091